reference/uitpas.json

```diff
@@ -2613,7 +2626,83 @@
             "CLIENT_ACCESS_TOKEN": []
           }
         ],
-        "description": "Retrieve information related to a pass.\n\nThe caller of this request must have `PASS_READ` permission."
+        "description": "Retrieve information related to a pass.\n\nIf the response contains `messages`, they MUST be displayed to the end-user.\n\nThe caller of this request must have `PASSES_READ` permission."
+      }
+    },
+    "/insz-numbers/{inszNumber}": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "inszNumber",
+          "in": "path",
+          "required": true
+        }
+      ],
+      "get": {
+        "summary": "Retrieve INSZ number",
+        "tags": [
+          "Passholders"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Pass"
+                },
+                "examples": {
```

Comment:
Nit: I think you can also include examples in `#/components/schemas/Pass` and the response will then "inherit" them if not overwritten here. So then you don't need to duplicate the examples for `/insz-numbers/{inszNumber}` and `/passes/{uitpasNumber}`

Reply:
stoplight doesn't seem to default to the examples in the model:
https://docs.publiq.be/docs/uitpas/branches/UPS-3826-passes-insz/b3A6NDg4OTQ4MzE-retrieve-insz-number

Reply:
See https://github.com/cultuurnet/stoplight-docs-uitpas/pull/64#discussion_r845167808